exampleplugins/5.2/SwagLastRegistrationsWidget/Resources/views/backend/index/swag_last_registrations/store/account.js

```diff
@@ -0,0 +1,42 @@
+
+
+Ext.define('Shopware.apps.Index.swagLastRegistrationsWidget.store.Account', {
+    /**
+     * Extends the default Ext Store
+     * @string
+     */
+    extend: 'Ext.data.Store',
```

Comment:
Shopware.store.Listing ?

if you change that, you need to rename your action in the backend controller to "listAction"